README.md

```diff
@@ -50,6 +50,15 @@ their [official documentation](https://github.com/CycloneDX/cyclonedx-property-t
             SBOM.</td>
         <td><code>metadata</code></td>
     </tr>
+    <tr style="vertical-align: top;">
+        <td><b>siemens:sbomNature</b></td>
+        <td>Used to indicate the nature of the entire SBOM document. Possible values are:
+            <ul><li><code>binary</code> – The SBOM contains binary components.</li>
+                <li><code>source</code> – The SBOM contains source components.</li>
+            </ul>
+            A binary component may have source information attached, and vice versa.</td>
```

Comment:
Would it make sense to mention *how* source information (or vice versa) should be attached?

Reply:
I added this sentence in order to avoid the misunderstanding that a binary or source SBOM can only contain information on binaries or sources, respectively. Each component can make full use of CycloneDX capabilities and contain both. Instead, the `sbomNature` property actually states how components were selected.

Reply:
Ok, removing the sentence is probably the best solution for now. Let's add more clarification once we see questions or discussions around it.